go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -694,15 +697,22 @@ func (bp *backupPipe) ReportProgress(period time.Duration, logger logutil.Logger
 	for {
 		select {
 		case <-bp.done:
-			logger.Infof("Done taking Backup %q", bp.filename)
+			n := float64(atomic.LoadInt64(&bp.nn))
+			logger.Infof("finished %s: %q (%.02fkb)", bp.action, bp.filename, n/1024.0)
 			return
 		case <-tick.C:
-			written := float64(atomic.LoadInt64(&bp.nn))
+			n := float64(atomic.LoadInt64(&bp.nn))
 			if bp.maxSize == 0 {
-				logger.Infof("Backup %q: %.02fkb", bp.filename, written/1024.0)
+				logger.Infof("%s %q: %.02fkb", bp.action, bp.filename, n/1024.0)
 			} else {
 				maxSize := float64(bp.maxSize)
-				logger.Infof("Backup %q: %.02f%% (%.02f/%.02fkb)", bp.filename, 100.0*written/maxSize, written/1024.0, maxSize/1024.0)
+				logger.Infof("%s %q: %.02f%% (%.02f/%.02fkb)",
+					bp.action,
+					bp.filename,
```

Comment:
I suspect that for restore this still prints out the encoded filename (just a monotonically increasing number) and not the real filename. Did that change in your testing?